main.py

```diff
@@ -1,10 +1,42 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        users = json.loads(file.read())
+        for nickname, params in users.items():
+            params_race = params["race"]
+            race, _ = Race.objects.get_or_create(
+                name=params_race["name"],
+                description=params_race["description"]
+            )
+            params_gild = params["guild"]
+            if params_gild:
+                guild, _ = Guild.objects.get_or_create(
+                    name=params_gild["name"],
+                    description=params_gild.get("description", "")
+                )
```

Comment:
Ensure that the `params_gild` dictionary contains a 'name' key to avoid KeyError exceptions. Consider using `params_gild.get('name')` with a default value or handling missing keys appropriately.